### DISCUSSION TITLE
Visibility of the "About" table field

### DISCUSSION THREAD

**Discussion by betterthanever2** [supabase/supabase#7484] [Questions]
Hello everyone. Is there a way to make the contents of the "about" field of a table visible in the Table Editor?

**Comment by GaryAustin1** [comment]
Are you talking about the description field when creating a table in the UI, or shown in the database table tab?
It is not in the Table Editor top level UI as far as I know.
If so you would have to suggest an enhancement in issues, or propose a PR to studio code to get it in the table editor.
